j2lint/linter/indenter/node.py

```diff
@@ -50,7 +51,8 @@ def create_node(self, line, line_no, indent_level=0):
         node.parent = self
         return node
 
-    def create_indentation_error(self, node, message):
+    @classmethod
+    def create_indentation_error(cls, node, message):
```

Comment:
Why make it a `@classmethod` - you can make it a `@staticmethod` here as yoy don;t need to `cls` argument:

```suggestion
    @staticmethod
    def create_indentation_error(node, message):
```